bench.c

```diff
@@ -20,10 +20,14 @@
  */
 #include "http_parser.h"
 #include <assert.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <string.h>
 #include <sys/time.h>
 
+/* 8 gb */
+static const int64_t kBytes = 8589934592LL;
```

Comment:
`8LL << 30`?  Little easier to parse for humans.

Reply:
Ack. PTAL